website_event_waiting_list/templates/waiting_list_registration.xml

```diff
@@ -17,12 +17,12 @@
                             <form action="/website_form/"
                                   method="post"
                                   data-model_name="crm.lead"
-                                  t-att-data-success_page="'%s/waiting_list_confirmation' % values.get('event').website_url"
+                                  t-att-data-success_page="'%s/waiting-list-confirmation' % event.website_url"
```

Comment:
Don't build URLs "manually" if you have them already -> `event.waiting_list_registration_url` ;)

Reply:
This URL is for the confirmation page, which isn't computed on the model.
'%s/waiting-list-confirmation' % event.website_url != event.waiting_list_registration_url

Reply:
and I am dumb :stuck_out_tongue_closed_eyes: :smile: